### ISSUE TITLE
pavucontrol-qt doesn't inherit system theme, icons and font

### ISSUE THREAD

**Issue by glu8716** [netblue30/firejail#6044]
### Description

I'm on LXQt and **pavucontrol-qt** doesn't inherit system theme, icons and font.

![1](https://github.com/netblue30/firejail/assets/33501455/941e7c1f-6713-413b-b5cf-657dad1b5bd9)

### Steps to Reproduce

1. Open **pavucontrol-qt** with Firejail on.

### Expected behavior

Program should inherit system theme, icons and font.

### Actual behavior

Program doesn't inherit system theme, icons and font.

### Behavior without a profile

Program has correct theme, font and icons.

![2](https://github.com/netblue30/firejail/assets/33501455/c682ad8d-fe8a-41cf-81fe-865b7fad62ce)

### Additional context

None.

### Environment

- Linux distribution and version: **Artix**
- Firejail version: **0.9.72**

### Checklist

- [X] The issues is caused by firejail (i.e. running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
- [X] I can reproduce the issue without custom modifications (e.g. globals.local).
- [X] The program has a profile. (If not, request one in `https://github.com/netblue30/firejail/issues/1139`)
- [X] The profile (and redirect profile if exists) hasn't already been fixed [upstream](https://github.com/netblue30/firejail/tree/master/etc).
- [X] I have performed a short search for similar issues (to avoid opening a duplicate).
  - [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
- [ ] I used `--profile=PROFILENAME` to set the right profile. (Only relevant for AppImages)

### Log

<details>
<summary>Output from the terminal</code></summary>
<p>

```
Reading profile /etc/firejail/pavucontrol-qt.profile
Reading profile /etc/firejail/pavucontrol.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/disable-exec.inc
Reading profile /etc/firejail/disable-interpreters.inc
Reading profile /etc/firejail/disable-programs.inc
Reading profile /etc/firejail/disable-xdg.inc
Reading profile /etc/firejail/whitelist-usr-share-common.inc
Reading profile /etc/firejail/whitelist-var-common.inc
Parent pid 7981, child pid 7982
1 program installed in 1.56 ms
Warning: An abstract unix socket for session D-BUS might still be available. Use --net or remove unix from --protocol set.
Warning: skipping alternatives for private /etc
Warning: skipping asound.conf for private /etc
Warning: skipping ld.so.preload for private /etc
Private /etc installed in 4.48 ms
Private /usr/etc installed in 0.01 ms
Warning: /sbin directory link was not blacklisted
Warning: /usr/sbin directory link was not blacklisted
Warning: not remounting /run/user/1000/gvfs
Warning: not remounting /run/user/1000/doc
Child process initialized in 64.64 ms

Parent is shutting down, bye...

```

</p>
</details>

**Comment by glitsj16** [comment]
There's a comment in the included `pavucontrol.profile` that might be relevant:

https://github.com/netblue30/firejail/blob/16edbd82687eb483038eb8749f972f8679fb1932/etc/profile-m-z/pavucontrol.profile#L18-L23

Yet we seem to have forgotten to do the same in `pavucontrol-qt.profile`:

https://github.com/netblue30/firejail/blob/16edbd82687eb483038eb8749f972f8679fb1932/etc/profile-m-z/pavucontrol-qt.profile#L12-L13

Not sure if this impacts `theming`, but it's something we'll need to correct regardless.

You can try again with the below and check if that works:

```sh
$ cat ~/.config/firejail/pavucontrol-qt.local
# Firejail profile for pavucontrol-qt
# Persistent local customizations

# whitelisting in ${HOME} is broken, see #3112
ignore mkdir ${HOME}/.config/pavucontrol-qt
ignore whitelist ${HOME}/.config/pavucontrol-qt

# try a blacklisting profile to fix theming
ignore whitelist /usr/share/pavucontrol
ignore whitelist /usr/share/pavucontrol-qt
ignore include whitelist-usr-share-common.inc
ignore include whitelist-var-common.inc

# try allowing D-bus session bus to fix theming
ignore dbus-user none
```

Also, do you use any of the Qt environment variables like QT_STYLE_OVERRIDE / QT_QPA_PLATFORMTHEME?

**Comment by glu8716** [comment]
> You can try again with the below and check if that works:

Yes it does! What makes the theme working for me is:
```
ignore mkdir ${HOME}/.config/pavucontrol-qt
ignore whitelist ${HOME}/.config/pavucontrol-qt
```
The others seem to make no difference.

> Also, do you use any of the Qt environment variables like QT_STYLE_OVERRIDE / QT_QPA_PLATFORMTHEME?

No I don't.

**Comment by glitsj16** [comment]
@glu8716

That's great news. It means my PR will be enough to fix this and keep whitelisting outside of ${HOME} and 'dbus-user none' intact. Just remember to drop this workaround when a next firejail release comes in via your OS.

Thanks again for reporting!

**Comment by glitsj16** [comment]
Fixed via #6045.
